servers/rendering/rendering_device.h

```diff
@@ -156,27 +156,33 @@ class RenderingDevice : public RenderingDeviceCommons {
 	//
 	// See the comments in the code to understand better how it works.
 
+	enum StagingRequiredAction {
+		STAGING_REQUIRED_ACTION_NONE,
+		STAGING_REQUIRED_ACTION_FLUSH_AND_STALL_ALL,
+		STAGING_REQUIRED_ACTION_STALL_PREVIOUS
```

Comment:
```suggestion
		STAGING_REQUIRED_ACTION_STALL_PREVIOUS,
```

Reply:
Fixed and pushed on Dario's behalf since he is on vacation